extensions/amp-script/validator-amp-script.protoascii

```diff
@@ -58,6 +58,7 @@ tags: {  # <amp-script>
   disallowed_ancestor: "AMP-SCRIPT"
   requires_extension: "amp-script"
   requires: "amp-experiment-token"
+  attrs: { name: "sandbox" }
```

Comment:
The comment in the .md file suggests that this can have a value, while this rule does not allow a value. Which is correct?

Reply:
We definitely need to allow values here, but they can be anything. This is the same way that `<amp-iframe>` specifies it's sandbox attribute

https://github.com/ampproject/amphtml/blob/4b86736870dc948e0c706fcac6317136d16af819/extensions/amp-iframe/validator-amp-iframe.protoascii#L56

Reply:
> this rule does not allow a value

I don't think this is true: https://github.com/ampproject/amphtml/pull/23143/files#diff-fe24fc57b0bb2f2b6bed265abb5c511bR47

Reply:
@Gregable friendly ping.